docs/docs/getting-started/config-options.mdx

```diff
@@ -113,3 +113,13 @@ translocoConfig({
   interpolation: ['<<<', '>>>']
 })
 ```
+
+### `scopes.keepNamespaceCasing`
+```ts
+translocoConfig({
+  // This will make sure that the casing of the provided scope name is not altered, given that no alias has been set otherwise this setting will be ignored
```

Comment:
Move this comment to be text under the title.

Reply:
Done